tokio/src/runtime/handle.rs

```diff
@@ -35,9 +35,30 @@ impl Handle {
             self.io_handle.clone(),
             self.time_handle.clone(),
             Some(self.clock.clone()),
+            Some(self.blocking_spawner.clone()),
         )
         .enter();
-        self.blocking_spawner.enter(|| f())
+
+        f()
+    }
+
+    /// Returns a Handle view over the currently running Runtime
+    ///
```

Comment:
Could I trouble you for a blurb on why/when you would need to use this function? Also, maybe a quick doc example? :heart:

Reply:
Is the new doc enough?

Reply:
Looks great :+1: